### ISSUE TITLE
Dummies guide

### ISSUE THREAD

**Issue by jrjhealey** [PokemonGoF/PokemonGo-Bot#2808]
# While the readmes etc are still out of date, and information is scattered all over various issues, I thought I'd share what exactly had worked for me:

Specs at my end (since it seems to be very dependent on what people are running):

## _On Linux_ I'm running:
`Ubuntu 14.04.5 LTS (GNU/Linux 3.13.0-65-generic x86_64)`
`pip 8.1.2`
`Python 2.7.6`
`virtualenv 15.0.2`
Fermuch git repo `153ea212999e337b54e09761f6defbabb0de4e8c`
My encrypt.so MD5 checksum (in case that helps):
`91d2eb79fcbc72b696f7523f93491325`

## _On OSX_ I'm running:
`OSX El Capitan 10.11.4 (15E65)`
`pip 8.1.2`
`Python 2.7.10`
`virtualenv 15.0.2`
Fermuch git repo `153ea212999e337b54e09761f6defbabb0de4e8c`
My encrypt.so MD5 checksum (in case that helps):
`MD5 (encrypt.so) = c715106d91e624728d6984ecfacefa14`

## What to do:
First I cloned fermuch's repo of the Bot:
`git clone --recursive https://github.com/fermuch/PokemonGo-Bot.git`

`cd PokemonGo-Bot/`

Run the included install script:
`bash install.sh`

I see no need to f*ck around with pip install etc, when people have written the scripts to do it!

Get the encrypt file, compile it and move it:
`wget http://pgoapi.com/pgoencrypt.tar.gz && tar -xf pgoencrypt.tar.gz && cd pgoencrypt/src/ && make && mv libencrypt.so ../../encrypt.so`

_For OSX:_
Try downloading the encrypt.so from http://www.mediafire.com/download/engnf5fvj65duxa/encrypt.so

or

`wget http://www.mediafire.com/download/engnf5fvj65duxa/encrypt.so`

Using this encrypt.so worked for me.

Set up your config file (edit it with your Google Maps API Key, logins etc.)

`cp ./configs/config.json.example ./configs/config.json`
`nano ./configs/config.json` #and do all the relevant editing.

Finally I ran it explicitly:
`python pokecli.py --config ./configs/config.json`

or `./run.sh configs/config.json`

On OSX, I had to invoke the bot with sudo to overcome a permissions error.

Hope that's useful for a few people.

**Comment by zetasq** [comment]
Dude, you can make a PR instead of issue :)

**Comment by jrjhealey** [comment]
Yeah I thought about that, but it seemed to me that most people are probably coming directly to the issues pages, so people might be more likely to spot it here and not open the same issue for the millionth time haha! Could do both I suppose.

**Comment by brantje** [comment]
Please send a PR =)

**Comment by sinthu94** [comment]
dude, are you able to catch pokemons?
cause i'm not, the bot just walk the whole time..

**Comment by jrjhealey** [comment]
I havent watched the bot for very long so far, but it says it has been capturing. 

Some example output:
`2016-08-07 13:47:19,476 [MoveToFort] [INFO] [moving_to_lured_fort] Moving towards pokestop University of Warwick - Science Park - 0.05km (attraction of lure 0.28km)
2016-08-07 13:47:26,788 [MoveToFort] [INFO] [moving_to_lured_fort] Moving towards pokestop University of Warwick - Science Park - 0.04km (attraction of lure 0.27km)
2016-08-07 13:47:33,765 [PokemonCatchWorker] [INFO] [pokemon_appeared] A wild Weedle appeared! [CP 77] [Potential 0.31] [S/A/D 3/9/2]
2016-08-07 13:47:36,874 [PokemonCatchWorker] [INFO] [threw_pokeball] Used Pokeball, with chance 53.30 (60 left)
2016-08-07 13:47:37,398 [PokemonCatchWorker] [INFO] [pokemon_caught] Captured Weedle! [CP 77] [Potential 0.31] [3/9/2] [+210 exp]`

**Comment by helmut1337** [comment]
The bot worked for a few hours, now I get this 

`2016-08-07 15:47:51,362 [       cli] [INFO] PokemonGO Bot v1.0
2016-08-07 15:47:51,362 [       cli] [INFO] No config argument specified, checking for /configs/config.json
2016-08-07 15:47:51,378 [       cli] [INFO] Configuration initialized
2016-08-07 15:47:51,378 [PokemonGoBot] [INFO] [set_start_location] Setting start location.
2016-08-07 15:47:51,378 [PokemonGoBot] [INFO] [x] Coordinates found in passed in location, not geocoding.
2016-08-07 15:47:51,378 [PokemonGoBot] [INFO] [location_found] Location found: 48.069478,9.951591 (48.069478, 9.951
591, 0.0)
2016-08-07 15:47:51,378 [PokemonGoBot] [INFO] [position_update] Now at (48.069478, 9.951591, 0)
2016-08-07 15:47:51,378 [PokemonGoBot] [INFO] [login_started] Login procedure started.
Traceback (most recent call last):
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokecli.py", line 493, in <module>
    main()
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokecli.py", line 127, in main
    report_summary(bot)
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokecli.py", line 136, in report_summary
    metrics.capture_stats()
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokemongo_bot\metrics.py", line 76, in capture_stats
    response_dict = request.call()
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokemongo_bot\api_wrapper.py", line 90, in call
    if not self.can_call():
  File "C:\Users\nutzer\Desktop\PokemonGo-Bot-master\pokemongo_bot\api_wrapper.py", line 58, in can_call
    raise NotLoggedInException()`

**Comment by jrjhealey** [comment]
Made a PR, but I've never actually done one on GH before so don't really know what I'm doing :P I suppose it should be merged in to the readme eventually, but I'm sure the devs will correct all the relevant details once the bot itself is finished being patched.

**Comment by jrjhealey** [comment]
@helmut1337 I would make a new issue or find an existing one with your issue so that this thread stays fairly clean. You're more likely to find an answer that way too.

**Comment by sinthu94** [comment]
@jrjhealey yeah, but seems like the bot still have a small issue related to catching pokemons..
(since the bot has been walking for 45min and catched 1 pokemon) :/

**Comment by alexanderrock** [comment]
Tried this on dev branch but i throws an error that it can't find the file encrypt.so
PokemonGo-Bot/encrypt.so

**Comment by sinthu94** [comment]
@alexanderrock please have a look on #2778

**Comment by jrjhealey** [comment]
@sinthu94 maybe take a look at the criteria you have for catchin pokemon? Maybe you could lower CP cutoffs etc to catch more.

@alexanderrock which repo did you clone from? is the encrypt.so file inside your main Bot folder?

**Comment by DustySky** [comment]
Thanks for this guide. You helped me to get it working. I even just made this account to say thank you.

**Comment by g2384** [comment]
This is a really clear guide. Thanks a lot.

This bot is the best. It has so many configurations and I have total control over my inventory.

**Comment by sinthu94** [comment]
@jrjhealey  i have the dafault settings...
"catch": {
      "any": {"catch_above_cp": 0, "catch_above_iv": 0, "logic": "or"},
      "// Example of always catching Rattata:": {},
      "// Rattata": { "always_catch" : true }
    },

**Comment by jrjhealey** [comment]
@sinthu94 hmm... I can only suppose it is where the bot is walking? I'm no dev so I don't know if the bot is performing as intended or not. My bot does seem to do a lot of walking, but it's caught some stuff too.

Have you tried changing your geolocation to somewhere more densely populated?

**Comment by alexanderrock** [comment]
@jrjhealey
I'mm testing both master and dev
for both i've written an installscript

installbot
`git clone -b master https://github.com/PokemonGoF/PokemonGo-Bot.git PokemonGo-Bot
cd PokemonGo-Bot
git pull
pip install -r requirements.txt --upgrade
wget http://pgoapi.com/pgoencrypt.tar.gz
tar -xf pgoencrypt.tar.gz
cd pgoencrypt/src/
make
mv libencrypt.so ../../encrypt.so
cd ../../
rm pgoencrypt.tar.gz
rm -rf pgoencrypt
chmod +x pokecli.py`

installbotdev
`git clone -b dev https://github.com/PokemonGoF/PokemonGo-Bot.git PokemonGo-Bot-dev
cd PokemonGo-Bot-dev
git pull
pip install -r requirements.txt --upgrade
wget http://pgoapi.com/pgoencrypt.tar.gz
tar -xf pgoencrypt.tar.gz
cd pgoencrypt/src/
make
mv libencrypt.so ../../encrypt.so
cd ../../
rm pgoencrypt.tar.gz
rm -rf pgoencrypt
chmod +x pokecli.py`

in both scenarios the encrypt.so is in the root of PokemonGo-Bot or PokemonGo-Bot-dev where the pokecli.py is too

the error is the following

> UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-07 16:15:18,645 [sentry.errors.uncaught] [ERROR] [u'OSError: encrypt.so: cannot open shared object file: No such file or directory', u'  File "pokecli.py", line 493, in <module>', u'  File "pokecli.py", line 127, in main', u'  File "pokecli.py", line 136, in report_summary', u'  File "pokemongo_bot/metrics.py", line 76, in capture_stats', u'  File "pokemongo_bot/api_wrapper.py", line 106, in call', u'  File "pokemongo_bot/api_wrapper.py", line 63, in _call', u'  File "pgoapi/pgoapi.py", line 210, in call', u'  File "pgoapi/rpc_api.py", line 85, in activate_signature', u'  File "ctypes/__init__.py", line 443, in LoadLibrary', u'  File "ctypes/__init__.py", line 365, in __init__']

**Comment by sinthu94** [comment]
@jrjhealey well, i'wll try with another geolocation, hopefully that'll work. thank you bro ;)

**Comment by jrjhealey** [comment]
@alexanderrock I'm afraid I've no insight on the official bot repo, but if you take another look at my OP, I'm currently using the version that's in fermuch's fork. I had issues with the official repo too but whatever fermuch has changed (specifically he made an edit to do with the file path of the encrypt.so file), seems to have made all the difference.

Consider cloning that repo and testing what I put in the OP and then let us know if that works.

**Comment by jrjhealey** [comment]
@sinthu94 word to the wise, don't jump too far in one go else you may trigger a softban ;)

**Comment by alexanderrock** [comment]
@jrjhealey 
Now its working with fermuchs fork

**Comment by Decubitus** [comment]
@jrjhealey @alexanderrock  could you help me to run the Bot? - I miss some basics to handle it alone :/

**Comment by Iamtheturtleking** [comment]
Hello, I have followed the instructons in the OP and I am getting this error.

Traceback (most recent call last):
  File "pokecli.py", line 38, in <module>
    from pgoapi.exceptions import NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineException
  File "/home/andrew/pokemongo-bot/src/pgoapi/pgoapi/__init__.py", line 50, in <module>
    from pgoapi.pgoapi import PGoApi
  File "/home/andrew/pokemongo-bot/src/pgoapi/pgoapi/pgoapi.py", line 34, in <module>
    from pgoapi.rpc_api import RpcApi
  File "/home/andrew/pokemongo-bot/src/pgoapi/pgoapi/rpc_api.py", line 46, in <module>
    from pgoapi.utilities import to_camel_case, get_time, get_format_time_diff, Rand48, long_to_bytes, generateLocation1, generateLocation2, generateRequestHash, f2i
  File "/home/andrew/pokemongo-bot/src/pgoapi/pgoapi/utilities.py", line 30, in <module>
    import xxhash
ImportError: No module named xxhash

**Comment by jrjhealey** [comment]
I don't know exactly what's causing your issue (other than you could try `pip2 install xxhash`) but the whole point here was so that people wouldn't have to mess around with installing things via pip explicitly.

Looking at what you've pasted though, your file path doesn't have capital letters in the Bot name, which is what you would get if you cloned the repo in the OP exactly as I have the command typed, so it looks to me like the instructions in the OP weren't followed _exactly_. 

Can you paste what your command line looked like and the exact command you used as well.

Consider doing this:

`cd ~/andrew/ && sudo rm -rf ~/andrew/pokemongo-bot/ && git clone --recursive https://github.com/fermuch/PokemonGo-Bot.git`

Do not make a folder for it is it will do that itself (unless you want to make a new folder, then `cd` in to it before you run the `git clone` command, then it will make a folder within a folder.

In the guide above, I've assumed everyone has followed the parts of the wiki which havent changed, i.e. having virtualenv, pip, python and so on already installed.

**Comment by Iamtheturtleking** [comment]
Thank you @jrjhealey 
pip2 install xxhash then prompted to use pip2 install raven and now it runs as it should

**Comment by as70873463** [comment]
OS : Windows 

how to fix ?

**Comment by jrjhealey** [comment]
I have no idea for Windows as I'm not using it. Hopefully some one who is and has it working can create a thread similar to this one with a Windows how-to

**Comment by g2384** [comment]
@as70873463 I am using [https://github.com/PokemonGoF/PokemonGo-Bot-Desktop](https://github.com/PokemonGoF/PokemonGo-Bot-Desktop) which has a visual map and is based on this repo. Follow its instruction to install all dependencies. 

After installation, you will see an error, please refer to this [https://github.com/PokemonGoF/PokemonGo-Bot-Desktop/issues/51](https://github.com/PokemonGoF/PokemonGo-Bot-Desktop/issues/51)

**Comment by Decubitus** [comment]
So im done with Windows :) working

**Comment by as70873463** [comment]
solution ? plz ^^ @Decubitus

**Comment by jrjhealey** [comment]
@Decubitus I suggest you make a new Issue thread specifically for Windows so that people can find it easy with a guide in your OP like this one

**Comment by Decubitus** [comment]
-> Download https://github.com/NECROBOTIO/NecroBot/releases/tag/v0.7.0
Start it and follow instrictions think its the easiest way

**Comment by Corecrafts** [comment]
@jrjhealey 

Keep gettin this error on mac

Traceback (most recent call last):
  File "pokecli.py", line 493, in <module>
    main()
  File "pokecli.py", line 129, in main
    raise e
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
2016-08-07 20:03:07,957 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/Users/nextlevelsam/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-07 20:03:07,958 [sentry.errors.uncaught] [ERROR] [u"UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)", u'  File "pokecli.py", line 493, in <module>', u'  File "pokecli.py", line 129, in main']

**Comment by yusong-shen** [comment]
Got the same error with @Corecrafts in Mac.

**Comment by jrjhealey** [comment]
I'll test the guide on my mac today and report back.  Only tested it on my Ubuntu VM so far.

**Comment by jrjhealey** [comment]
@Core crafts and @yusong-shen I would guess based on that output that there is an issue with your encrypt.so compiling on Mac. 

Have a look through the issues for an encrypt.so thread about Mac. I'm sure I've seen them but not currently at a computer to browse efficiently.

**Comment by yusong-shen** [comment]
Hi @jrjhealey , thanks for your reply. 
For me yesterday, the bot worked well for about half an hour, then I encountered above error. When I restarted it last night, this time it run entire night without crash, but it had same issue as you mention in #3062 , the bot only gain 6k xp in about 8.5 hour which is about 20 minutes actual work. I set the location in Union Square ,SF.

**Comment by jrjhealey** [comment]
Just had a go at installing as above on my mac and there is definitely an issue when it comes to compiling the encrypt.so for me - I don't have a solution yet so if anyone knows, do say:

`...
               ~~~~~^~~~~~~ ~
encrypt.c:13621:14: note: place parentheses around the '&' expression to silence this warning
        v411 = v392 & ~v385 | v47;
                    ^
               (           )
encrypt.c:13966:47: warning: '&' within '|' [-Wbitwise-op-parentheses]
        v50 = (v19 & v4 ^ *(_DWORD *)(result + 452)) & ~v30 | *(_DWORD *)(result + 416);
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~ ~
encrypt.c:13966:47: note: place parentheses around the '&' expression to silence this warning
        v50 = (v19 & v4 ^ *(_DWORD *)(result + 452)) & ~v30 | *(_DWORD *)(result + 416);
                                                     ^
              (                                            )
66 warnings generated.
cc -shared -o libencrypt.so encrypt.o -lm
cc -fPIC -std=gnu99   -c -o main.o main.c
cc -o test -L. main.o -lencrypt
`

**Comment by jrjhealey** [comment]
Possible fix for OSX:

using this encrypt.so: http://www.mediafire.com/download/engnf5fvj65duxa/encrypt.so
Reported in #2746 and #2861 got me past the issue mentioned before.

I now have the following error:
`Traceback (most recent call last):
  File "pokecli.py", line 493, in <module>
    main()
  File "pokecli.py", line 129, in main
    raise e
IOError: [Errno 13] Permission denied: u'web/inventory-**********@gmail.com.json'`

Though this might be due to a softban I'm currently having.

**Comment by jrjhealey** [comment]
Ok, latest update, bot is now working on my OSX installation. I had to invoke it with sudo to overcome the last error bizarrely but hey..whatever works..

Key seemed to be downloading the pre-compiled encrypt.so

**Comment by youyi1314** [comment]
for Window User:
1. Download: https://github.com/PokemonGoMap/PokemonGo-Map/blob/develop/pogom/encrypt32bit.dll
2. Move file to **x:\path\to\PokemonGo-Bot\**
2. Rename to **encrypt.dll**
3. Run 

> python pokecli.py

 and Enjoy

**Comment by Tcheno** [comment]
I got the following error on install.sh just now:
Command "/var/www/pokego/dev/PokemonGo-Bot/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-ewivVR/numpy/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-zjZ6J1-record/install-record.txt --single-version-externally-managed --compile --install-headers /var/www/pokego/dev/PokemonGo-Bot/include/site/python2.7/numpy" failed with error code 1 in /tmp/pip-build-ewivVR/numpy/

**Comment by jrjhealey** [comment]
@Tcheno can you post your python, virtualenv and pip versions

**Comment by Tcheno** [comment]
I'm runnig on debian jesse. Versions are as follows:
Python 2.7.9
Virtualenv 15.0.3
pip 8.1.2 from /usr/local/lib/python2.7/dist-packages (python 2.7)

**Comment by Tcheno** [comment]
more on the error:

running build_py
    copying numpy/version.py -> build/lib.linux-armv7l-2.7/numpy
    copying build/src.linux-armv7l-2.7/numpy/__config__.py -> build/lib.linux-armv7l-2.7/numpy
    copying build/src.linux-armv7l-2.7/numpy/distutils/__config__.py -> build/lib.linux-armv7l-2.7/numpy/distutils
    running build_clib
    customize UnixCCompiler
    customize UnixCCompiler using build_clib
    running build_ext
    customize UnixCCompiler
    customize UnixCCompiler using build_ext
    building 'numpy.core.umath' extension
    compiling C sources
    C compiler: arm-linux-gnueabihf-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC

    compile options: '-DHAVE_NPY_CONFIG_H=1 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -Ibuild/src.linux-armv7l-2.7/numpy/core/src/umath -Inumpy/core/include -Ibuild/src.linux-armv7l-2.7/numpy/core/include/numpy -Inumpy/core/src/private -Inumpy/core/src -Inumpy/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inumpy/core/src/umath -Inumpy/core/src/npysort -I/usr/include/python2.7 -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -c'
    arm-linux-gnueabihf-gcc: numpy/core/src/umath/ufunc_type_resolution.c
    arm-linux-gnueabihf-gcc: numpy/core/src/umath/ufunc_object.c
    arm-linux-gnueabihf-gcc: numpy/core/src/umath/umathmodule.c
    arm-linux-gnueabihf-gcc: numpy/core/src/umath/reduction.c
    arm-linux-gnueabihf-gcc: build/src.linux-armv7l-2.7/numpy/core/src/umath/scalarmath.c
    arm-linux-gnueabihf-gcc: build/src.linux-armv7l-2.7/numpy/core/src/umath/loops.c
    numpy/core/src/umath/loops.c.src: In function âSHORT_subtractâ:
    numpy/core/src/umath/loops.c.src:844:1: internal compiler error: Segmentation fault
     }
     ^
    Please submit a full bug report,
    with preprocessed source if appropriate.
    See <file:///usr/share/doc/gcc-4.9/README.Bugs> for instructions.
    Preprocessed source stored into /tmp/ccMA2tE3.out file, please attach this to your bugreport.
    numpy/core/src/umath/loops.c.src: In function âSHORT_subtractâ:
    numpy/core/src/umath/loops.c.src:844:1: internal compiler error: Segmentation fault
     }
     ^
    Please submit a full bug report,
    with preprocessed source if appropriate.
    See <file:///usr/share/doc/gcc-4.9/README.Bugs> for instructions.
    Preprocessed source stored into /tmp/ccMA2tE3.out file, please attach this to your bugreport.
    error: Command "arm-linux-gnueabihf-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -DHAVE_NPY_CONFIG_H=1 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE=1 -D_LARGEFILE64_SOURCE=1 -Ibuild/src.linux-armv7l-2.7/numpy/core/src/umath -Inumpy/core/include -Ibuild/src.linux-armv7l-2.7/numpy/core/include/numpy -Inumpy/core/src/private -Inumpy/core/src -Inumpy/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inumpy/core/src/umath -Inumpy/core/src/npysort -I/usr/include/python2.7 -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -Ibuild/src.linux-armv7l-2.7/numpy/core/src/private -c build/src.linux-armv7l-2.7/numpy/core/src/umath/loops.c -o build/temp.linux-armv7l-2.7/build/src.linux-armv7l-2.7/numpy/core/src/umath/loops.o" failed with exit status 1

**Comment by k4n30** [comment]
@jrjhealey - Can this cluster issue be closed?

**Comment by jrjhealey** [comment]
Yes I assume so. Thread has been quiet for a few days now
